script/module/CellMap.js

```diff
@@ -0,0 +1,112 @@
+import Cell from 'module/Cell';
+
+export default class CellMap extends Array {
+    static createWithSize(size) {
+        let cellMap = [];
+
+        for (let x = 0; x < size.width; x += 1) {
+            cellMap[x] = [];
+
+            for (let y = 0; y < size.height; y += 1) {
+                cellMap[x][y] = new Cell({x: x, y: y});
+            }
+        }
+
+        return new this(cellMap);
+    }
+
+    constructor(cells) {
+        super(...cells);
+    }
+
+    forEachCell(callback) {
```

Comment:
Je kunt ook overwegen om op de een of andere manier een array met alle cellen toegankelijk te maken. `getCells` of zo, en dan kun je daar gewoon alle array methods op gebruiken zonder er helper methods voor te maken.

Of zelfs een computed property:
```
Object.assign(this, {
    get cells() {
        return this.reduce((cells, row) => cells.concat(row), []));
    }
});
```

Zoiets misschien (niet getest hoor)

Reply:
Lekker hoor, dit idee werkt supergoed.